cmd/buddha.go

```diff
@@ -194,6 +196,7 @@ func runJob(job *buddha.Job) error {
 				continue
 			}
 		}
+		cmd.Failures = failures
```

Comment:
Rather than replacing and re-setting, it would be cleaner and not much more invasive to make `executeChecks` take the number of failures as a parameter, and pass either `1` for before checks or `cmd.Failures` for after checks...